impl/src/main/java/org/jboss/cdi/tck/tests/lookup/circular/DependentSelfConsumingNormalProducer.java

```diff
@@ -1 +1 @@
-/*
 * JBoss, Home of Professional Open Source
 * Copyright 2010, Red Hat, Inc., and individual contributors
 * by the @authors tag. See the copyright.txt in the distribution for a
 * full listing of individual contributors.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 * http://www.apache.org/licenses/LICENSE-2.0
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,  
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package org.jboss.cdi.tck.tests.lookup.circular;

import jakarta.enterprise.context.ApplicationScoped;
import jakarta.enterprise.inject.Produces;
import jakarta.inject.Inject;

public class DependentSelfConsumingNormalProducer {
    @Inject
    @SelfConsumingNormal1
    Violation violation;

    @Produces
    @ApplicationScoped
    @SelfConsumingNormal1
    public Violation produceViolation() {
        return new Violation(DependentSelfConsumingNormalProducer.class.getName());
    }

    public void ping() {
        violation.ping();
    }
}
\ No newline at end of file
+/*
 * JBoss, Home of Professional Open Source
 * Copyright 2010, Red Hat, Inc., and individual contributors
 * by the @authors tag. See the copyright.txt in the distribution for a
 * full listing of individual contributors.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 * http://www.apache.org/licenses/LICENSE-2.0
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,  
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package org.jboss.cdi.tck.tests.lookup.circular;

import jakarta.enterprise.context.ApplicationScoped;
import jakarta.enterprise.context.Dependent;
import jakarta.enterprise.inject.Produces;
import jakarta.inject.Inject;

@Dependent
public class DependentSelfConsumingNormalProducer {
    @Inject
    @SelfConsumingNormal1
    Violation violation;

    @Produces
    @ApplicationScoped
    @SelfConsumingNormal1
    public Violation produceViolation() {
        return new Violation(DependentSelfConsumingNormalProducer.class.getName());
    }

    public void ping() {
        violation.ping();
    }
}
```

Comment:
This file looks really odd :-)

Reply:
Uum, what the heck? I only added `@Dependent` here :-D

Strange thing is, this looks normal in my IDE. It also looks normal in the [raw form on GH](https://raw.githubusercontent.com/eclipse-ee4j/cdi-tck/master/impl/src/main/java/org/jboss/cdi/tck/tests/lookup/circular/DependentSelfConsumingNormalProducer.java).
But it does look mangled when you just [browse the file](https://github.com/eclipse-ee4j/cdi-tck/blob/master/impl/src/main/java/org/jboss/cdi/tck/tests/lookup/circular/DependentSelfConsumingNormalProducer.java). And that was the case even before my commit.
Honestly, no idea how to fix that.

Reply:
Not your fault, it was like that before and IDEs are smart. It looks like the line endings in the file are CR only (not CRLF like Windows, and not LF only like Unix), which AFAIK was only used on Mac OS up to version 9! :laughing: